packages/permission-log-controller/src/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './enums';
```

Comment:
Looking at the extension, it seems that these constants aren't used outside of the PermissionsController, so do you think we need to export them. (They _are_ used in tests, but tests directly import this file, so removing this line shouldn't break the tests.)

``` suggestion
```